File: dab2c-executor-application/src/main/java/ru/sbrf/ufs/dab2c/core/ApplicationEntryPoint.java
package ru.sbrf.ufs.dab2c.core;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jms.JmsAutoConfiguration;
import org.springframework.boot.autoconfigure.validation.ValidationAutoConfiguration;
import org.springframework.context.annotation.EnableAspectJAutoProxy;

/**
 * Входная точка для запуска Spring-Boot-приложения.
 */
@EnableAspectJAutoProxy
@SpringBootApplication(exclude = {JmsAutoConfiguration.class, ValidationAutoConfiguration.class})
public class ApplicationEntryPoint {

    /**
     * Запуск Spring-приложения
     * @param args Аргументы
     */
    public static void main(String[] args) {
        SpringApplication.run(ApplicationEntryPoint.class, args);
    }
}

File: executor-application/src/main/kotlin/ru/sbrf/dab2c/executor/application/ApplicationEntryPoint.kt
package ru.sbrf.dab2c.executor.application

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan

/** Spring Boot application entry point. */
@SpringBootApplication
@ComponentScan(basePackages = ["ru.sbrf.dab2c.executor"])
class ApplicationEntryPoint

/** Starts the Spring application. */
@Suppress("SpreadOperator")
fun main(args: Array<String>) {
    SpringApplication.run(ApplicationEntryPoint::class.java, *args)
}
